open-api/clients/javascript-client/.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
```

Comment:
Do we cirlcle ci or travis for continous integration? 
Which system does which part?